pyatlan/client/asset.py

```diff
@@ -2029,6 +2030,31 @@ def process_assets(
                     func(asset)
         return len(guids_processed)
 
+    @validate_arguments
+    def add_or_update_dq_rules_schedule(
```

Comment:
This will be consistent with the existing workflow schedule addition method:
```py
  def add_schedule(
        self,
        workflow: Union[
            WorkflowResponse,
            WorkflowPackage,
            WorkflowSearchResult,
            WorkflowSearchResultDetail,
        ],
        workflow_schedule: WorkflowSchedule,
    ) -> WorkflowResponse:
```
```suggestion
    def add_dq_rule_schedule(
```